source/includes/_datasets.md

```diff
@@ -105,12 +104,52 @@ This endpoint lists all the datasets that can be edited by this user.
 Parameter | Default | Description
 --------- | ------- | -----------
 `where` <br> *string* | None | Filter expression used to restrict returned datasets ([ODSQL documentation](https://docs.opendatasoft.com/api/explore/v2.html#where-clause))
-`start` <br> *string* | 0 | Index of the first item to return
+`page` <br> *string* | 1 | Number of the page you want to retrieve. Pages numbers vary when changing the number of rows to retrieve.
```

Comment:
The 2nd part of the description confused me, I had to reread it several times to understand it. I think we can consider people know what a page is, no?

Reply:
No problem, I'll drop it